sriov/tests/SR_IOV_macAddress_DPDK/test_SR_IOV_macAddress_DPDK.py

```diff
@@ -20,15 +20,22 @@ def test_SR_IOV_macAddress_DPDK(dut, trafficgen, settings, testdata):
     dut_ip = testdata['dut_ip']
     vf0_mac = testdata['dut_mac']
     pf = settings.config["dut"]["interface"]["pf1"]["name"]
+
     steps = [
-        "echo 0 > /sys/class/net/{}/device/sriov_numvfs".format(pf),
-        "echo 1 > /sys/class/net/{}/device/sriov_numvfs".format(pf),
-        "ip link set {} vf 0 mac {}".format(pf, vf0_mac),      
+       "echo 0 > /sys/class/net/{}/device/sriov_numvfs".format(pf),
+       "echo 1 > /sys/class/net/{}/device/sriov_numvfs".format(pf),   
         ]
     for step in steps:
         code, out, err = dut.execute(step)
         assert code == 0, err
     
+    assert vfs_created(dut, pf, 1, 3)
```

Comment:
no need to use "3" second, the default value (10) is what openshift uses and we should stick to it; also, we may want to add an assert a message to explain what went wrong, so something like this will be better:  assert vfs_created(dut, pf, 1), f"Failed to create VF on {pf}"